aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -10,6 +10,17 @@
 namespace at { namespace native {
 
 namespace {
+#if AT_BUILD_WITH_LAPACK()
+char to_lapack(TransposeType trans) {
```

Comment:
nit: The same characters are used in BLAS. It should be either `to_blas` or `to_blaslapack`.